pkgs/tools/system/netdata/default.nix

```
@@ -23,6 +23,11 @@ stdenv.mkDerivation rec{
       $out/libexec/netdata/plugins.d/apps.plugin.org
   '';
 
+  preConfigure = ''
+    substituteInPlace collectors/python.d.plugin/python_modules/third_party/lm_sensors.py \
+      --replace 'ctypes.util.find_library("sensors")' "\"${lm_sensors.out}/lib/libsensors.so\""
```

To make it work on macOS as well:

```suggestion
      --replace 'ctypes.util.find_library("sensors")' "\"${lm_sensors.out}/lib/libsensors.${stdenv.hostPlatform.extensions.sharedLibrary}\""
```

---

done !